docs/tutorials/device/nodeaffinitylabels.md

```diff
@@ -1,11 +1,11 @@
-# Use a unique Node-Selector label
+# Use Node-Selector labels
 
-Hostpath LocalPV uses the Kubernetes Node label `kubernetes.io/hostname=<node-name>` to uniquely identifly a node.
+device LocalPV uses the Kubernetes Node label(s) (For example: `kubernetes.io/hostname=<node-name>`) to uniquely identifly a node.
 
 In some cases, this label (`hostname`) is not unique across all the nodes in the cluster. This was seen on clusters provisioned with [Bosh](https://bosh.io/docs/) across different fault domains.
 
-A unique Node label may be used instead of the above mentioned Kubernetes default label to uniquely identify a node. This label may be set by you, the administrator.
-This label can be set when defining a StorageClass. One such sample StorageClass is given below...
+A unique Node label or set of labels may be used instead of the above mentioned Kubernetes default label to uniquely identify a node. This label(s) may be set by you, the administrator.
+This label(s) can be set when defining a StorageClass. One such sample StorageClass is given below...
```

Comment:
A unique Node label or set of labels may be used  --> A unique Node label (or set of labels) may be used